### DISCUSSION TITLE
How to get the strikethrough symbol of a font in excel?

### DISCUSSION THREAD

**Discussion by niuzhi** [qax-os/excelize#1450] [General]
How to get the strikethrough symbol of a font in excel?
![image](https://user-images.githubusercontent.com/33443724/212632455-bb61274e-e35f-4827-a025-40e6d84312a0.png)
![image](https://user-images.githubusercontent.com/33443724/212632520-12ad3831-0efe-45c9-8507-a8838735015f.png)

**Comment by xuri** [comment]
Thanks for your feedback. Please don't create the same topic twice. This library support setting the strikethrough style for the cell, but doesn't support getting style details currently. This is a duplicate of issue #314.
